server/.firebaserc

```diff
@@ -0,0 +1,6 @@
+{
+  "projects": {
+    "dev": "who-mh2-dev1",
```

Comment:
Should we force someone to select dev0, dev1 or others specifically? If we have an alias for dev, then that'll be the one that gets all the collisions.

Reply:
If at some point we introduce a `dev2` project we can remove the alias. Odds are we may only ever have one active `dev` project though.